# Lab | Inferential statistics
## Instructions

1. It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of `130.1 mm Hg` with a standard deviation of `21.21 mm Hg`. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

    * Set up the hypothesis test.
    * Write down all the steps followed for setting up the test.
    * Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.
    

2. If you finished the previous question, please go through the code for `principal_component_analysis_example` provided in the `files_for_lab folder`.

In [ ]:
import math

### 1. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

In [37]:
population_mean = 120
n = 100
sample_mean = 130.1
sample_std = 21.21

df = n-1    #degree of freedom

se = sample_std/math.sqrt(n)    # standard error

#### Calculating our t-statistics

In [40]:
statistic = (sample_mean - population_mean)/se
print("The test statistic is: ", statistic)

The test statistic is:  4.761904761904759


**The test statistic value of 4.76 being > t.95=1.660 rejects the Null Hypothesis with a p-value of 5%. So it would be very suprisingly, if the diseased patients would have the same low mean systolic blood pressure.**

### 2. If you finished the previous question, please go through the code for `principal_component_analysis_example` provided in the `files_for_lab folder`.